email_notification/helpers/email_builder_reservation.py

```diff
@@ -221,20 +229,23 @@ def _get_my_reservations_ext_link(language: LanguageType | None) -> str:
             url_base = urljoin(url_base, language) + "/"
```

Comment:
Above: `_get_my_reservations_ext_link` `language` parameter is not nullable.

Reply:
And looks like it is never `None` either, fixed the typing